mep/people/views.py

```diff
@@ -29,7 +29,14 @@ class MembersList(ListView):
     def get_queryset(self):
         # limit to people with library accounts, do not include
         # item creators who are not library members
-        return super().get_queryset().exclude(account=None)
+        return super().get_queryset().exclude(account=None).exclude(mep_id='')
+
+
+class MemberDetail(DetailView):
+    model = Person
+    template_name = 'people/member_detail.html'
+    context_object_name = 'member'
+    slug_field = 'mep_id'
```

Comment:
This view should 404 on attempt to view a non-library member. I think if you override the `get_queryset` and apply that filter it will work. We might want to implement a custom queryset with a `library_member` filter, we're using that multiple places now - although it doesn't necessarily have to be done for this feature.